src/dashboard/menu/menuItem/styles.ts

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components';
+import $ListItem from "@material-ui/core/ListItem";
+import { ElementType } from 'react';
+
+type ListItemType = {
+  isDisabled?: boolean;
+  component: ElementType;
+  button: boolean;
+  to: string;
+  selected: boolean
+}
+
+export const ListItem = styled($ListItem)<ListItemType>`
+  ${p => p.isDisabled && `
```

Comment:
Currently there's a console error in dev mode. 

> Warning: React does not recognize the `isDisabled` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `isdisabled` instead. If you accidentally passed it from a parent component, remove it from the DOM element.

Not sure if it's necessary to add it to the DOM but if it's ok and there's no other good workaround, I suggest changing it to `disabled` instead.